renovate.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": [
-    "config:base"
+    "config:base",
```

Comment:
```suggestion
```

This is already included in https://github.com/powerhome/renovate-config/blob/main/default.json#L3